src/main/scala/firrtl/transforms/FixAddingNegativeLiteralsTransform.scala

```diff
@@ -0,0 +1,106 @@
+// See LICENSE for license details.
+
+package firrtl.transforms
+
+import firrtl.{CircuitState, LowForm, Namespace, PrimOps, Transform, Utils, WRef}
+import firrtl.ir._
+import firrtl.Mappers._
+import firrtl.PrimOps.{Add, AsSInt, Sub, Tail}
+
+import scala.collection.mutable
+
+object FixAddingNegativeLiterals {
+
+  /** Returns the maximum negative number represented by given width
+    * @param width width of the negative number
+    * @return maximum negative number
+    */
+  def maxNegNumber(width: BigInt): BigInt = -(1 << (width.toInt - 1))
```

Comment:
Thanks for the Scaladoc; I've often looked for intmin/intmax functions, so this is definitely reuse- and doc-worthy, but could there be some confusion with `INT_MIN` or other "min" kinds of names for this?

Reply:
Do you have a suggestion for a replacement name?

Reply:
`minNegValue` or `minValue`?